astropy/modeling/mappings.py

```diff
@@ -10,16 +10,34 @@
 
 
 class Mapping(Model):
-    def __init__(self, mapping):
+    """
+    Allows inputs to be reordered, duplicated or dropped.
+
+    Parameters
+    ----------
+    mapping : tuple
+        Integers representing indices of the inputs.
+
+    Raises
+    ------
+    TypeError
+        Raised when number of inputs is less that `max(mapping)`.
```

Comment:
The max(mapping) should have double-backticks (without them this is going to fail with a warning in the doc build).